definitions/procedures/restore/configs.rb

```diff
@@ -35,11 +40,17 @@ def restore_configs(backup)
 
       feature(:tar).run(tar_options)
     end
+    # rubocop:enable  Metrics/MethodLength
 
     def reload_configs
       feature(:mongo).reload_db_config if feature(:mongo)
     end
 
+    def online_backup?
+      metadata_file = YAML.load(File.read(options['backup_dir'] + 'metadata.yml'))
+      @online_flag = metadata_file['online']
+    end
+
```

Comment:
This method should be in https://github.com/theforeman/foreman_maintain/blob/master/lib/foreman_maintain/utils/backup.rb - the backup instance is already created with `backup = ForemanMaintain::Utils::Backup.new(@backup_dir)`. You could either add a new method there or check the existing ones and see if the could help you.